pyproject.toml

```diff
@@ -48,7 +48,6 @@ ignore = [
     "SIM102", "SIM103", "SIM112", # flake8-simplify code styles
     # these ignores are from flake8-simplify. please fix or ignore with commented reason
     "SIM108",
-    "SIM109",
```

Comment:
Also please edit the .flake8 config to remove the ignore from there as well.

Reply:
Done